client/src/components/GlobalStyles.js

```diff
@@ -11,11 +13,21 @@ function GlobalStyles() {
         *:after {
           box-sizing: border-box;
         }
+        @font-face {
+            font-family: 'Quicksand Light';
+            src: url('${QuicksandLight}') format('opentype');
+          }
+          @font-face {
+            font-family: 'Quicksand Book';
+            src: url('${QuicksandBook}') format('opentype');
+          }
+          @font-face {
+            font-family: 'Rubik';
+            src: url('${Rubik}') format('truetype');
+          }
         body {
           height: 100vh;
-          font-size: 34;
```

Comment:
you deleted the `font-size` here. if you use `rem` in your components, it's needed to set a default `font-size` in the body like 17px because `rem` orientates after that...